test/core/api/service/chain/chain_api_test.cpp

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+
+#include "api/service/chain/impl/chain_api_impl.hpp"
+#include "mock/core/blockchain/block_tree_mock.hpp"
+#include "mock/core/blockchain/header_repository_mock.hpp"
+#include "primitives/block_header.hpp"
+#include "testutil/literals.hpp"
+#include "testutil/outcome.hpp"
+
+using kagome::api::ChainApiImpl;
+using kagome::blockchain::BlockTreeMock;
+using kagome::blockchain::HeaderRepositoryMock;
+using kagome::common::Buffer;
+using kagome::primitives::BlockHash;
+using kagome::primitives::BlockHeader;
+using kagome::primitives::BlockInfo;
+using testing::Return;
+
+/**
+ * @given chain api
+ * @when get a block hash value without parameter
+ * @then last finalized block hash value is returned
+ */
+TEST(ChainApiTest, GetBlockHashNoParam) {
+  auto block_header_repo = std::make_shared<HeaderRepositoryMock>();
+  auto block_tree = std::make_shared<BlockTreeMock>();
+
+  ChainApiImpl api{block_header_repo, block_tree};
```

Comment:
You do this in every test, why not just create fixture and do that during SetUp?

Reply:
refactored